ui/src/pages/backoffice/Location/LocationForms/InternalLocationEditor/components/InternalLocationForm/InternalLocationForm.js

```diff
@@ -28,12 +28,16 @@ export class InternalLocationForm extends Component {
     ]);
   };
 
-  updateInternalLocation = (pid, data) => {
-    return internalLocationApi.update(pid, data);
+  updateInternalLocation = async (pid, data) => {
+    const response = await internalLocationApi.update(pid, data);
+    await delay();
```

Comment:
While here we inject the delay in the component, in the `internalLocation.delete` the delay is part of the actions which I think is the place we should put all the delays in order to avoid the pitfall of having them twice (not sure if this is the case anywhere).

Reply:
I checked the code and there are reducers only for `locationList` and `internalLocationList` so, the components is the only place.

Reply:
Discussed by voice.